app/experimenter/experiments/tests/factories.py

```diff
@@ -216,6 +220,69 @@ def projects(self, create, extracted, **kwargs):
         self.projects.add(*extracted)
 
 
+class ExperimentRapidFactory(ExperimentConstants, factory.django.DjangoModelFactory):
+    type = Experiment.TYPE_RAPID
```

Comment:
Why is there a type here?

Reply:
This is built on the current model definitions, which requires the type be set.  We can take it out in the split pr.

Reply:
Nevermind. :)